src/main/java/entities/Post.java

```diff
@@ -0,0 +1,53 @@
+package entities;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+/**
+ * A Post that Users make on Foodstagram.
+ */
+public class Post {
+    private ArrayList<User> likedUsers;
```

Comment:
I think we should add getters and setters for all these instance variables. Otherwise looks good!